### ISSUE TITLE
Wallabag doesn't appear to work

### ISSUE THREAD

**Issue by benjamin051000** [Lissy93/portainer-templates#4]
When I run the Wallabag app template, I get this error:
```
  curl_multi_init           OK!                                                                                                                         
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
14:21:53 ERROR     [console] Error thrown while running command "wallabag:install --env=prod -n". Message: "Some system requirements are not fulfilled. Please check output messages and fix them." ["exception" => RuntimeException { …},"command" => "wallabag:install --env=prod -n","message" => "Some system requirements are not fulfilled. Please check output messages and fix them."]
In InstallCommand.php line 163:
                                                                               
  Some system requirements are not fulfilled. Please check output messages an  
  d fix them.                                                                  
                                                                               
wallabag:install [--reset] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
Starting Wallabag ...
Configuring the SQLite database ...
wallabag installer
==================
Step 1 of 4: Checking system requirements.
------------------------------------------
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
  Checked                   Status   Recommendation                                                                                                     
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
  PDO Driver (pdo_sqlite)   OK!                                                                                                                         
  Database connection       ERROR!   Can't connect to the database: An exception occurred in driver: SQLSTATE[HY000] [14] unable to open database file  
  Database version          OK!                                                                                                                         
  curl_exec                 OK!                                                                                                                         
  curl_multi_init           OK!                                                                                                                         
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
14:22:46 ERROR     [console] Error thrown while running command "wallabag:install --env=prod -n". Message: "Some system requirements are not fulfilled. Please check output messages and fix them." ["exception" => RuntimeException { …},"command" => "wallabag:install --env=prod -n","message" => "Some system requirements are not fulfilled. Please check output messages and fix them."]
In InstallCommand.php line 163:
                                                                               
  Some system requirements are not fulfilled. Please check output messages an  
  d fix them.                                                                  
                                                                               
wallabag:install [--reset] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
Starting Wallabag ...
Configuring the SQLite database ...
wallabag installer
==================
Step 1 of 4: Checking system requirements.
------------------------------------------
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
  Checked                   Status   Recommendation                                                                                                     
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
  PDO Driver (pdo_sqlite)   OK!                                                                                                                         
  Database connection       ERROR!   Can't connect to the database: An exception occurred in driver: SQLSTATE[HY000] [14] unable to open database file  
  Database version          OK!                                                                                                                         
  curl_exec                 OK!                                                                                                                         
  curl_multi_init           OK!                                                                                                                         
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
14:23:47 ERROR     [console] Error thrown while running command "wallabag:install --env=prod -n". Message: "Some system requirements are not fulfilled. Please check output messages and fix them." ["exception" => RuntimeException { …},"command" => "wallabag:install --env=prod -n","message" => "Some system requirements are not fulfilled. Please check output messages and fix them."]
In InstallCommand.php line 163:
                                                                               
  Some system requirements are not fulfilled. Please check output messages an  
  d fix them.                                                                  
                                                                               
wallabag:install [--reset] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
Starting Wallabag ...
Configuring the SQLite database ...
wallabag installer
==================
Step 1 of 4: Checking system requirements.
------------------------------------------
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
  Checked                   Status   Recommendation                                                                                                     
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
  PDO Driver (pdo_sqlite)   OK!                                                                                                                         
  Database connection       ERROR!   Can't connect to the database: An exception occurred in driver: SQLSTATE[HY000] [14] unable to open database file  
  Database version          OK!                                                                                                                         
  curl_exec                 OK!                                                                                                                         
  curl_multi_init           OK!                                                                                                                         
 ------------------------- -------- ------------------------------------------------------------------------------------------------------------------- 
14:24:48 ERROR     [console] Error thrown while running command "wallabag:install --env=prod -n". Message: "Some system requirements are not fulfilled. Please check output messages and fix them." ["exception" => RuntimeException { …},"command" => "wallabag:install --env=prod -n","message" => "Some system requirements are not fulfilled. Please check output messages and fix them."]
In InstallCommand.php line 163:
                                                                               
  Some system requirements are not fulfilled. Please check output messages an  
  d fix them.                                                                  
                                                                               
wallabag:install [--reset] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
```

This is likely because Wallabag expects a DB connection. See their official docker compose: https://github.com/wallabag/docker#docker-compose

**Comment by hacketaler** [comment]
the wallabag-template does not work. db-connection and db-tables are not correct.

**Comment by shmolf** [comment]
tldr;

It looks like this is an application issue, and not a Docker configuration issue.
[This was fixed in June of 2023](https://github.com/wallabag/docker/issues/333#issuecomment-1587023269)

Either way, this project is not responsible for correcting misconfigurations with Docker apps. This project only aggregates the apps from other sources.

---

This github project doesn't create the configuration files. Instead, this project aggregates the docker configurations from other projects.

This app, `wallabag`, seems to originate from [this other project](https://github.com/xneo1/portainer_templates/blob/72c14ca2584024b91f9335e709943264f4e1dcf7/Template/template.json#L3740). But this is not a guarantee, since the docker configurations are loaded as first-come-first-stored, and another project may actually be responsible for this issue.

Assuming the project linked above is the correct one, and looking at that [project's stackfile](https://github.com/xneo1/portainer_templates/blob/master/Template/Stack/wallabag.yml), there is a database configured.
- Except some minor differences (like the `port`).

**Comment by shmolf** [comment]
See the recently updated [app list](https://github.com/Lissy93/portainer-templates?tab=readme-ov-file#supported-apps-and-stacks) with a new `Report Issue` link next to each app.

**Comment by lissy93** [comment]
This repo is just an aggregator, so any issues need to be reported in the template source.
See: https://github.com/lissy93/portainer-templates/blob/main/.github/BROKEN_APP.md

I will work on a CI to try and identify broken apps, and prevent them getting pulled in, but in the meantime use the Report Issue button next to a given app.
